exporter/redis.go

```diff
@@ -116,6 +116,8 @@ var (
 
 	instanceInfoFields = map[string]bool{"role": true, "redis_version": true, "redis_build_id": true, "redis_mode": true, "os": true}
 	slaveInfoFields    = map[string]bool{"master_host": true, "master_port": true, "slave_read_only": true}
+
+	dataBasesCount = 0
```

Comment:
Not sure this will work if you keep per-redis-instance state in one global variable. If the exporter is used to scrape more than one instance then they'll get into each others way.

Reply:
It's a bit cumbersome but a way to solve this would be to have `extractConfigMetrics()` return `(int, error)` and you return the dbCount there so you can pass to `extractInfoMetrics()`

Reply:
Thank you. Fixed it